### DISCUSSION TITLE
How to generate API for dotnet core 2.1 ?

### DISCUSSION THREAD

**Discussion by LucaGabi** [Dotnet-Boxed/Templates#316] [Q&A]
﻿**Which project template is the issue about?**
API
...

**Which version of the project template is the issue about?**
3.2.0
...

**Which version of Visual Studio or the dotnet SDK were you using?**
2.1.2
...

**Comment by VictorioBerra** [comment]
You might be able to install an older version of the template from NuGet and use dotnet new. But why would you want to do that? Just use 2.2. There are many bug fixes, performance and security improvements in both aspnet core and this template.

**Comment by RehanSaeed** [comment]
We only support the latest version of .NET Core. Any more than that is too much overhead to support.

**Comment by davihar** [comment]
I had the same need so I forked the repo and removed .NET Core 2.2 specific features:

https://github.com/davihar/Framework

Why do this?  I'm using IoT Edge and it currently only supports .NET Core 2.1.  Someday when it supports 2.2, I'll stop using my fork.
